css/style.css

```diff
@@ -0,0 +1,99 @@
+body {
+  background-color: #EEE;
+  padding-left: 30px;
+  padding-right: 30px;
+}
+
+h1 {
+  color: red;
+  font-family: cursive, sans-serif;
+  text-align: center;
+  font-size: 32px;
+}
+
+h2 {
+  color: maroon;
+  font-family: cursive, sans-serif;
+  text-align: center;
+}
+
+p {
+  color: black;
+  font-family: fantasy, sans-serif;
+  text-align: left;
+}
+
+img {
+  border: 10px solid;
+  padding: 5px;
+  margin: 0px 60px 10px 60px;
+  height: 400px;
+  float: right;
+}
+
+.portfolio {
+  border: 0px;
+  margin: 0px 40px 10px 40px;
+  padding: 0px;
+  height: 500px;
+  float: none;
+}
+
+img.portrait {
+  float: none;
+  border: 2px;
+  margin: 0px;
```

Comment:
When using 0 values for rules, omit the unit of measure as it is assumed that the value is zero in all units. 🔨